pinot-core/src/main/java/org/apache/pinot/core/minion/SegmentConverter.java

```diff
@@ -121,9 +119,10 @@ public SegmentConverter(List<File> inputIndexDirs, File workingDir, String table
       }
 
       // Sorting on sorted column and creating indices
-      if (_indexingConfig != null) {
-        List<String> sortedColumn = _indexingConfig.getSortedColumn();
-        List<String> invertedIndexColumns = _indexingConfig.getInvertedIndexColumns();
+      if (_tableConfig != null) {
```

Comment:
Table config cannot be null, and we can remove this check

Reply:
removed